cmd/flow-capture.go

```diff
@@ -91,7 +169,24 @@ func runFlowCaptureOnAddr(addr string, filename string) {
 		log.Errorf("Create file %s failed: %v", filename, err.Error())
 		log.Fatal(err)
 	}
-	defer CleanupCapture(conn, f)
+	// Open SQLite database
+	var db *sql.DB
+	db, err = sql.Open("sqlite3", "./messages.db")
+	if err != nil {
+		log.Errorf("Error opening database: %v", err.Error())
+		log.Fatal(err)
+	}
+	defer CleanupCapture(conn, f, db)
+	// Create messages table if not exists
+	_, err = db.Exec(`CREATE TABLE IF NOT EXISTS messages (
+        id INTEGER PRIMARY KEY,
+        message TEXT
```

Comment:
I guess you will need to define every fields here to be efficient ?

Reply:
yeah I was just experminting I will just start with drops